src/graph.cpp

```diff
@@ -7,21 +7,42 @@
 

```

Comment:
Overall, I would like to see more documentation within the class methods.

Reply:
Have you seen graph.hpp?